test/e2e/config/e2e_conf.yaml

```diff
@@ -53,6 +53,7 @@ providers:
         contract: v1beta1
         files:
           - sourcePath: "../data/infrastructure-oci/v1beta1/cluster-template.yaml"
+          - sourcePath: "../data/infrastructure-oci/v1beta1/cluster-template-alternative-region.yaml"
```

Comment:
can you please add the test output of the new test please

Reply:
```
• [SLOW TEST:1115.007 seconds]
Workload cluster creation
/Users/jkratzat/Projects/cluster-api-provider-oci-joekr/test/e2e/cluster_test.go:49
  when Multi-Region workload cluster creation
  /Users/jkratzat/Projects/cluster-api-provider-oci-joekr/test/e2e/cluster_test.go:399
    Alternative region - With 1 control-plane nodes and 1 worker nodes
    /Users/jkratzat/Projects/cluster-api-provider-oci-joekr/test/e2e/cluster_test.go:401
------------------------------
STEP: Tearing down the management cluster


Ran 1 of 17 Specs in 1254.717 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 16 Skipped
```